Modules/selectmodule.c

```diff
@@ -1415,17 +1416,13 @@ pyepoll_internal_ctl(int epfd, int op, int fd, unsigned int events)
     case EPOLL_CTL_MOD:
         ev.events = events;
         ev.data.fd = fd;
-        Py_BEGIN_ALLOW_THREADS
         result = epoll_ctl(epfd, op, fd, &ev);
-        Py_END_ALLOW_THREADS
```

Comment:
As far as I can tell `epoll_ctl` operations like this always complete quickly, so releasing the GIL here isn't really necessary.